### PR TITLE
Bug 1603241 - Accessibility improvements in Intermittent Failures View

### PR DESCRIPTION
**Pull request by yogmel**
## Description of issue

- [X] **Table**: role should be set to `table`;
- [X] **Search inputs**: need `aria-label`;
- [X] **Row - details link**: show link when row/cell is focused;
- [X] **Previous/Next button**: set focus style

## Proposed Solution

For the attribute insertion, they needed to be passed down by ReactTable props, such as `getTableProps`. The passed function is declared in `ui/intermittent-failures/helpers.js`.

The other tasks were solved via CSS.

## Testing
Simple NVDA navigation and visual test on the CSS related tasks. Other views were also tested - only one view has the same React Table component, Perfherder Health, but apparently they are not linked.

### CODE REVIEW

**Comment by sarah-clements** [comment]
@yogmel I'm going to see if @camd can review this one - that's why I removed myself and added him! :)

**Comment by yogmel** [comment]
@sarah-clements so sorry!! I thought **I** had forgotten to add you.

**Comment by sarah-clements** [comment]
> @sarah-clements so sorry!! I thought **I** had forgotten to add you.

No worries :)

File: ui/intermittent-failures/helpers.js [hunk 1]
```diff
@@ -120,4 +120,24 @@ export const tableRowStyling = function tableRowStyling(state, bug) {
   return {};
 };
 
+export const tableProps = function tableProps() {
```
Comment (camd) [inline ui/intermittent-failures/helpers.js]: Do we plan on using these functions in other classes?  If not, why not just create them in the ``MainView.jsx`` file?  Or even create them inline at the definition of ``ReactTable``.  I think that would create less noise, tbh.

Comment (yogmel) [reply ui/intermittent-failures/helpers.js]: Sure, I have moved them to the `MainView.jsx` file. :)

File: ui/intermittent-failures/MainView.jsx [hunk 2]
```diff
@@ -105,6 +105,16 @@ const MainView = props => {
     } = calculateMetrics(graphData));
   }
 
+  const theadFilterThProps = (state, bug, data) => {
```
Comment (sarah-clements) [inline ui/intermittent-failures/MainView.jsx]: Just a fly-by comment :) This function name is a bit confusing/vague. Could you rename it so its purpose is clearer? Something like `getHeaderAriaLabel`.

Comment (yogmel) [reply ui/intermittent-failures/MainView.jsx]: Sure, it's done ;)

**Pull request merged**